agasc/tests/test_agasc_2.py

```diff
@@ -641,3 +642,10 @@ def test_write_missing_column(tmp_path, stars_in):
 
     # OK for full_agasc=False
     write_agasc(temp, stars=stars, version="test", full_agasc=False)
+
+
+def test_last_updated():
+    last_updated = agasc.get_supplement_table("last_updated")
+    assert CxoTime(last_updated["supplement"]).date.startswith("20")
```

Comment:
I see that there is no other testing of these supplement tables. So while you are at it, it would be worth testing:
- `last_updated` has the correct four keys
- Each corresponding value is a string that is a valid ISO format date `CxoTime(last_updated[key], format="iso")`.
- `agasc_versions` has the same four keys.

The name of the test function does not quite match what it does.